contracts/connectors/loantoken/LoanTokenLogicWeth_Arbitrum.sol

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2017-2021, bZxDao. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "./LoanTokenLogicStandard.sol";
+import "../../interfaces/IaeWETH.sol";
+
+
+contract LoanTokenLogicWeth_Arbitrum is LoanTokenLogicStandard {
```

Comment:
there's some changes in the code (really just in dealing with WETH), so let me know if you have questions. You guys will need to keep that in mind for sweep fees or other contract that withdraw from WETH later on